aws-codeartifact-repository/.rpdk-config

```diff
@@ -0,0 +1,17 @@
+{
+    "typeName": "AWS::CodeArtifact::Repository",
+    "language": "java",
+    "runtime": "java8",
```

Comment:
Are we updating this to a newer version of Java?

Reply:
I'll create an issue to track this